src/main/webapp/app/shared/consistency-check/consistency-check.component.html

```diff
@@ -0,0 +1,50 @@
+<div>
+    <div class="modal-header">
+        <h4 class="modal-title" jhiTranslate="artemisApp.consistencyCheck.title">Consistency checks</h4>
+        <button type="button" class="btn-close" data-dismiss="modal" aria-hidden="true" (click)="closeModal()"></button>
+    </div>
+    <div class="modal-body">
+        <div class="justify-content-center p-1">
+            <jhi-alert></jhi-alert>
+            <jhi-alert-error></jhi-alert-error>
+            <span jhiTranslate="artemisApp.consistencyCheck.text">Checks on programming exercises are performed and occurring consistency errors will be displayed below.</span>
+            <div *ngIf="isLoadingResults()" class="d-flex justify-content-center m-5">
+                <div class="spinner-border" role="status">
+                    <span class="sr-only">{{ 'loading' | artemisTranslate }}</span>
+                </div>
+            </div>
+            <div class="table-responsive p-1 my-5" *ngIf="!isLoadingResults()">
+                <table class="table table-striped">
+                    <thead>
+                        <tr>
+                            <th><span jhiTranslate="artemisApp.programmingExercise.detail.title">Programming Exercise</span></th>
+                            <th><span jhiTranslate="artemisApp.consistencyCheck.error.type">Error type</span></th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <tr *ngFor="let error of errors">
+                            <td>
+                                <a
+                                    *ngIf="courseId != undefined"
+                                    (click)="closeModal()"
+                                    [routerLink]="['/course-management', courseId, 'programming-exercises', error.programmingExercise!.id]"
```

Comment:
Correct :slightly_smiling_face: 

I added this with the consistency check of courses in mind, as it might be useful for the user to navigate directly to the inconsistent exercise.

In case of checks on individual programming exercises this loses its usefulness, I agree..

Reply:
But for courses, there is no link at all right? It's basically just the other way around then :D
But I agree, for courses it makes sense.